### PR TITLE
chore(content-picker): Cypress tests for content picker selection

### PR DESCRIPTION
**Pull request by ConradJChan**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/elements/content-picker/Footer.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const Footer = ({
 }: Props) => (
     <footer className="bcp-footer">
         <div className="bcp-footer-left">
-            <Button className="bcp-selected" onClick={onSelectedClick} type="button">
+            <Button className="bcp-selected" onClick={onSelectedClick} type="button" data-testid="bcp-selected">
```
Comment (jstoffan) [inline src/elements/content-picker/Footer.js]: Can we select this by its content, instead? Using `data-testid` is kind of a fallback solution.

File: test/integration/content-picker/ContentPicker.e2e.test.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    load(additionalProps = {}) {
+        cy.visit('/Elements/ContentPicker', {
+            onBeforeLoad: contentWindow => {
+                contentWindow.PROPS = additionalProps;
+            },
+        });
+
+        cy.getByTestId('be-sub-header').contains('Codepen');
+    },
+    getContentRow: rowNum => cy.getByTestId('content-picker').find(`.bcp-item-row-${rowNum}`),
```
Comment (jstoffan) [inline test/integration/content-picker/ContentPicker.e2e.test.js]: Should this be `getRow` to match `selectRow` and `unselectRow` below?

**Pull request merged**